### DISCUSSION TITLE
How to disable .py1.stats files in disk

### DISCUSSION THREAD

**Discussion by chanfried** [w0rp/ale#3803] [Q&A - Ask for help with problems]
vim 8.2, when ale plugin enabled
when opening python files(*.py),   A *.py1.stats file will be created in py file's location for each python file.  how to  disable  .py1.stats files

![image](https://user-images.githubusercontent.com/1514920/124378559-90819e00-dce4-11eb-94df-ded976b0b472.png)

**Comment by bersbersbers** [comment]
Try https://stackoverflow.com/questions/66677007/file-stats-when-run-python-in-neovim

**Comment by chanfried** [comment]
got it,thanks.
